docs/src/acceleration.md

```diff
@@ -0,0 +1,35 @@
+# Acceleration and parallelism
+
+!!! warning "Experimental API"
+
+    The acceleration API is experimental and may not work correctly in all
+    cases, especially if trying to use an acceleration method that your
+    version of Julia or installed packages cannot support. The API is also
+    subject to breaking changes during minor or major releases without
+    warning.
+
+## User-facing interface
+
```

Comment:
There is a style guide for the docs :-)  . Can you do ### instead of ## here. Otherwise we get an entry in the contents pane, which I reserve for the case of very long subsections.